projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-actions/order-detail-actions.component.html

```diff
@@ -19,6 +19,7 @@
           } | cxUrl
         "
         class="btn btn-block btn-action"
+        data-cy-cancel-button="cancelButton"
```

Comment:
explained how in the order-cancellations.ts in comments

Reply:
[Best Practices with Cypress?](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements) I can still take it out and use the other method if it's preferred

Reply:
It is what Cypress said to do when using selectors, but is it what `Spartacus` wants? Have you discussed that we pollute our tags with attributes solely for cypress?  If not, we need to talk to the tribe about it, because we would need to re-configure our e2e architecture and gear our templates to be e2e friendly